project/src/main/webapp/script.js

```diff
@@ -0,0 +1,85 @@
+var map;
+var autocomplete;
+var photoURLS; 
+var image;
+
+function execute() {
+    initMap();
+    search();
+    getCatalog();
+}
+
+function initMap() {
```

Comment:
Isn't the map functionality handled by me, is there an alternative purpose for this?

Reply:
just for me to visualiize